### DISCUSSION TITLE
Build error with oneAPI tbb package

### DISCUSSION THREAD

**Discussion by A2va** [xmake-io/xmake-repo#1568] [Q&A]
I am making a xmake package and it depends on oneAPI library, and therefore on tbb. There are build issue with xmake but not with cmake, it's weird.

The xmake log are the result of `xmake lua .\scripts\test.lua -v -D tbb`. Cmake log are when I built myself.

[cmake_log.txt](https://github.com/xmake-io/xmake-repo/files/9846406/cmake_log.txt)
[xmake_log.txt](https://github.com/xmake-io/xmake-repo/files/9846407/xmake_log.txt)

**Comment by waruqi** [comment]
> LINK : fatal error LNK1181: cannot open input file 'Leresche\AppData\Local\.xmake\cache\packages\2210\t\tbb\2021.6.0\source\src\tbb\def\win64-tbb.def' [C:\Users\Antoine Leresche\AppData\Local\.xmake\cache\packages\2210\t\tbb\2021.6.0\source\build_ae3a831b\src\tbb\tbb.vcxproj]

it's just tbb issue, you can patch and fix it and open a pr.

**Reply by A2va** [reply]
This is getting weirder, ttb compile on my laptop but not on my PC. I have xmake v2.7.2+HEAD.d88193698 on both of them.

**Reply by waruqi** [reply]
I works on ci. https://github.com/xmake-io/xmake-repo/actions/runs/3309479736

**Reply by A2va** [reply]
Gotcha I found it , it was some space in the build path. I change `XMAKE_GLOBALDIR` to set another location. Thank you for your help !
